src/dsp_tools/resources/start-stack/docker-compose.yml

```diff
@@ -4,31 +4,37 @@ version: "3.7"
 
 services:
 
-  app:
```

Comment:
I changed the order, so that it corresponds to https://github.com/dasch-swiss/ops-deploy/blob/main/roles/dsp-deploy/files/RELEASE.json. So, in the future, the copy-pasting should become easier.